vm/devices/get/get_protocol/src/dps_json.rs

```diff
@@ -178,6 +181,23 @@ pub struct HclDevicePlatformSettingsV2DynamicSmbios {
     pub processor_upgrade: u8,
 }
 
+#[derive(Debug, Default, Deserialize, Serialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct HclDevicePlatformSettingsAcpiTables {
```

Comment:
Same here, can't we just have an array of arbitrary tables?

Reply:
This seems doable, but UEFI still needs to know which is which somehow

Reply:
I have updated the host-side PR to do this.  As a result, I think the config.rs change will need to go through the tables and match them up to config that UEFI can take